S2VX.Game/EndGame/UserInterface/ScoreGrade.cs

```diff
@@ -0,0 +1,59 @@
+﻿using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics;
+
+namespace S2VX.Game.EndGame.UserInterface {
+    public class ScoreGrade : CompositeDrawable {
+        public string Grade { get; private set; } = "F";
+        public bool IsSquared { get; private set; }
+
+        public ScoreGrade(double accuracy, bool isFullCombo) {
+            DetermineGrade(accuracy, isFullCombo);
+
+            Origin = Anchor.Centre;
+            Position = new(250);
+            Size = new(500);
```

Comment:
Can we use constants rather than magic numbers for positioning/sizing?

Reply:
Changed to use local variables. Though as far as readability goes, I feel like I may prefer the magic numbers here. It's harder now to see relationships since I have to look up the variables for reference.